spec/features/courses/requirement_spec.rb

```diff
@@ -1,21 +1,28 @@
 require 'rails_helper'
 
 feature 'Course requirements', type: :feature do
-  let(:course_jsonapi) do
-    jsonapi(
-      :course,
-      provider: jsonapi(:provider, provider_code: 'A0')
-    )
+  let(:course_1) { jsonapi :course, name: 'English', provider: provider, include_nulls: [:accrediting_provider] }
+  let(:course_2) { jsonapi :course, name: 'Biology', include_nulls: [:accrediting_provider] }
+  let(:course_3) { jsonapi :course, name: 'Physics', include_nulls: [:accrediting_provider] }
+  let(:course_4) { jsonapi :course, name: 'Science', include_nulls: [:accrediting_provider] }
```

Comment:
We're not actually using `courses` anywhere except to assign them to the provider. Can we just use `provider: provider` in each of `course_2`, `course_3` and `course_4`?

Reply:
Come to think of it, why _do_ we generate so many courses here?